src/code/z_actor.c

```diff
@@ -4432,13 +4432,16 @@ void Animation_ChangeByInfo(SkelAnime* skelAnime, AnimationInfo* animationInfo,
                      frameCount, animationInfo->mode, animationInfo->morphFrames);
 }
 
-void func_80034F54(PlayState* play, s16* arg1, s16* arg2, s32 arg3) {
+/*
+ * computes `.limbOverrides` values for `*_OverrideLimbDraw` functions
+ */
+void UpdateLimbOverrides(PlayState* play, s16* tableY, s16* tableZ, s32 count) {
```

Comment:
Just for your future reference, we follow the structure of `SystemName_FunctionName` for all of our functions. In this case the system name would be `Actor_` and the function name would be `_UpdateLimbOverrides`

Ofc this function name is already changing, as discussed in discord, but wanted you to be aware for future function renames.

Reply:
good, noted. the fact that this function doesn't really have to do with actors in the first place is what confused me. besides `z64actor.h` doesn't follow the mentioned convention anyway. huh, well, probably it's a subject to a refactor then.

Reply:
"this function doesn't really have to do with actors in the first place"
It definitely does, actors are what use this system in the first place to rotate their joints

"z64actor.h doesn't follow the mentioned convention anyway"
The system name does not necessarily equate to the filename or the header name. There are many different subsystems contained within the broader actor system.

Its a bit subjective since there is no explicit "namespace" concept like you have in other language. But we make the determiniation based on the types of things the functions operate on and their relation to other functions that make up the concept of a "system" working together to achieve a certain task.

In this case, it seems fine to say its a part of the actor system because it is updating actor joints to simulate an idle animation for them. But you can argue it should be a part of a different system or even its own system.

Reply:
great! as always, noted

> Its a bit subjective
yeah, it is

again, i have no problems with C's "lack" of namespaces
- translation units are quite a "namespaces" themselves
- it's a good heuristic to put a system-related API to a respective header
  ```
    // could have gone to a `z64gfx.h` / `z_actor.c`, as it is now, or `z_actor_gfx.c`
    Gfx_DrawDListOpa
  ```
- not ideal to expose, say, `LinkAnimation_Update` to all users of `z64animation.h`
  ```
    // could have gone to a `z64link.h` / `z_skelanime.c`, as it is now, or `z_skelanime_link.c`
    // than only `z_en_torch2.c` and `z_player.c` would include it
    LinkAnimation_Update
  ```

but i can imagine matching decomp/recomp challenges with piling up `*.c` files and shuffling stuff around - not sure, but as an assumption. i have no complete context for that.

it do not urge you to spill the beans right now, merely conveying my thought process and vision on C generally. however it be, i got your point and will follow the convention `%system%_%function%`